UnityProject/Assets/Scripts/Items/Weapons/Gun.cs

```diff
@@ -130,10 +130,10 @@ protected StandardProgressActionConfig ProgressConfig
 		public AddressableAudioSource DryFireSound;
 
 		/// <summary>
-		/// The amount of times per second this weapon can fire
+		/// The time in seconds before this weapon can fire again.
 		/// </summary>
-		[Tooltip("The amount of times per second this weapon can fire")]
-		public double FireRate;
+		[Tooltip("The time in seconds before this weapon can fire again.")]
+		public double FireDelay = 0;
```

Comment:
is zero a good default delay?

Reply:
Honestly, no. Will change to 0.5 right now (which is similar to how fast guns with 0 delay fire in /tg/ due to lag and stuff, actually).